systemtest/src/test/java/io/strimzi/systemtest/Resources.java

```diff
@@ -706,6 +707,70 @@ KubernetesClusterRoleBindingBuilder defaultKubernetesClusterRoleBinding(String y
                 .endSubject();
     }
 
+    List<KubernetesClusterRoleBinding> clusterRoleBindingsForAllNamespaces(String namespace) {
```

Comment:
Why are you creating new `KubernetesClusterRoleBinding()` instead of using default bindings and change what you need? Bindings are loaded from file so we can ensure that we are testing files which are shipped.

Reply:
We don't have default bindings for CO to watch all namespaces